### PR TITLE
Address text changes in the Connections role mapping UI

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- https://github.com/wso2/product-is/issues/19428

### Related PRs
- https://github.com/wso2/identity-apps/pull/5515

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx [hunk 1]
```diff
@@ -196,14 +196,32 @@ export const IdentityProviderGroupsTab: FunctionComponent<IdentityProviderGroups
...
+                        i18nKey={
+                            "extensions:console.identityProviderGroups.claimConfigs.groupAttributeMessageSAML"
+                        }
+                        tOptions={ {
+                            attribute: groupClaim
+                        } }
+                    >
+                        Please note that <strong>{ groupClaim }</strong> attribute will be
+                        considered as the default <strong>Group Attribute</strong> as you have not added
+                        a custom attribute mapping for the connection&apos;s group attribute.
```
Comment (NipuniBhagya) [inline apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx:223]:
```suggestion
                        a custom attribute mapping.
```

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx [hunk 2]
```diff
@@ -196,14 +196,32 @@ export const IdentityProviderGroupsTab: FunctionComponent<IdentityProviderGroups
...
+                        i18nKey={
+                            "extensions:console.identityProviderGroups.claimConfigs.groupAttributeMessageOIDC"
+                        }
+                        tOptions={ {
+                            attribute: groupClaim
+                        } }
+                    >
+                        Please note that OpenID Connect attribute <strong>{ groupClaim }</strong> will be
+                        considered as the default <strong>Group Attribute</strong> as you have not added
+                        a custom attribute mapping for the connection&apos;s group attribute.
```
Comment (NipuniBhagya) [inline apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx:210]:
```suggestion
                        a custom attribute mapping.
```

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx [hunk 3]
```diff
@@ -196,14 +196,32 @@ export const IdentityProviderGroupsTab: FunctionComponent<IdentityProviderGroups
...
+                content={ isOIDC ? (
+                    <Trans
+                        i18nKey={
+                            "extensions:console.identityProviderGroups.claimConfigs.groupAttributeMessageOIDC"
+                        }
+                        tOptions={ {
+                            attribute: groupClaim
+                        } }
+                    >
+                        Please note that OpenID Connect attribute <strong>{ groupClaim }</strong> will be
```
Comment (NipuniBhagya) [inline apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx:208]:
```suggestion
                        Please note that OpenID Connect attribute named <strong>{ groupClaim }</strong> will be
```

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx]: Fixed with 139483e295b15b3fbecb20747c554a54facbb0f2

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx (hunk 2 above)
Comment (DonOmalVindula) [reply apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx]: Fixed with 139483e295b15b3fbecb20747c554a54facbb0f2

File: apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx (hunk 3 above)
Comment (DonOmalVindula) [reply apps/console/src/features/connections/components/edit/settings/identity-provider-groups/identity-provider-groups.tsx]: Fixed with 139483e295b15b3fbecb20747c554a54facbb0f2

File: apps/console/src/features/connections/components/edit/settings/attribute-management/uri-attributes-settings.tsx [hunk 4]
```diff
@@ -119,6 +119,32 @@ export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPr
...
+                                            "forms.uriAttributeSettings.group.messageOIDC"
+                                    }
+                                    tOptions={ {
+                                        attribute: ConnectionManagementConstants.OIDC_ROLES_CLAIM
+                                    } }
+                                >
+                                    Please note that OpenID Connect attribute named
+                                    <strong>{ ConnectionManagementConstants.OIDC_ROLES_CLAIM }</strong>
+                                    will be considered as the default <strong>Group Attribute</strong>
+                                    as you have not added a custom attribute mapping
```
Comment (NipuniBhagya) [inline apps/console/src/features/connections/components/edit/settings/attribute-management/uri-attributes-settings.tsx:140]:
```suggestion
                                    as you have not added a custom attribute mapping.
```

File: apps/console/src/features/connections/components/edit/settings/attribute-management/uri-attributes-settings.tsx [hunk 5]
```diff
@@ -211,6 +231,32 @@ export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPr
...
+                                        "forms.uriAttributeSettings.group.messageOIDC"
+                                }
+                                tOptions={ {
+                                    attribute: ConnectionManagementConstants.OIDC_ROLES_CLAIM
+                                } }
+                            >
+                                Please note that OpenID Connect attribute named
+                                <strong>{ ConnectionManagementConstants.OIDC_ROLES_CLAIM }</strong>
+                                will be considered as the default <strong>Group Attribute</strong>
+                                as you have not added a custom attribute mapping
```
Comment (NipuniBhagya) [inline apps/console/src/features/connections/components/edit/settings/attribute-management/uri-attributes-settings.tsx:252]:
```suggestion
                                as you have not added a custom attribute mapping.
```

**Review by NipuniBhagya — APPROVED** [review]

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**